models/dt-multiplelinearregression/src/main.py

```diff
@@ -0,0 +1,68 @@
+from enfobench.dataset import Dataset
+import pandas as pd
+import numpy as np
+from darts import TimeSeries
+from darts.models import RegressionModel
+from sklearn.linear_model import BayesianRidge,LinearRegression
+from enfobench.evaluation import evaluate_metrics
+from enfobench.evaluation.metrics import mean_absolute_error, mean_bias_error,root_mean_squared_error
+from enfobench import AuthorInfo, ModelInfo, ForecasterType
+from enfobench.evaluation.server import server_factory
+from enfobench.evaluation.utils import create_forecast_index, periods_in_duration
+from enfobench.dataset.utils import create_perfect_forecasts_from_covariates
+
+class MultipleLinearRegressionDarts:
+
+    def __init__(self, seasonality: str):
+        self.seasonality = seasonality()

```

Comment:
You are calling a string parameter.

```python
def __init__(self, seasonality: str):
        self.seasonality = seasonality
```

Reply:
maybe add .upper() to the seasonality like this

```python
def __init__(self, seasonality: str):
        self.seasonality = seasonality.upper()
```

Reply:
You are calling a string parameter.

How should i define the seasonality?

Reply:
I am using pandas period strings to define seasonality such as `1D` for 1 day etc.

Reply:
I have fixed this, and the code should be working now.